### ISSUE TITLE
Local build fails for the GATE Cloud Client

### ISSUE THREAD

**Issue by johann-petrak** [GateNLP/cloud-client#2]
When running "mvn install" (Maven 3.5.2) in the root directory, the following error occurs:

```
[ERROR] Failed to execute goal on project gate-cloud-plugin: Could not resolve dependencies for project uk.ac.gate.plugins:gate-cloud-plugin:jar:1.2-SNAPSHOT: 
Failure to find uk.ac.gate:gate-core:jar:1.2-SNAPSHOT in http://repo.gate.ac.uk/content/groups/public/ was cached in the local repository, resolution will not be reattempted until the update interval of gate-repo has elapsed or updates are forced -> [Help 1]
```

**Comment by greenwoodma** [comment]
Looks like your local cache is corrupt, and for some reason it's looking for 1.2-SNAPSHOT of gate-core which it will never find. Try deleting your cache and trying again

**Comment by ianroberts** [comment]
This is probably due to the change in `gate-plugin-base` to separate the version of plugin base from the version of gate-core, and should be fixable by adding a property setting to the pom.

**Comment by greenwoodma** [comment]
Ah yes I should revert that as it's completely broken

**Comment by johann-petrak** [comment]
Not sure I understand what the issue is an how to fix it? Is it something that can be done within this project?

**Comment by ianroberts** [comment]
GateNLP/gate-top@abc4f95d1ca092e05721a4cc3c1ca8c8386f9bd2 makes it so that the gate-core version is no longer tied to that of gate-plugin-base but instead comes from a property set in the plugin’s pom. The default is `${project.version}` which works nicely for core plugins where the version of the plugin generally follows that of core but in this case it means it’ll default to 1.2-SNAPSHOT. The fix is to add a `gate-core.version` property to the cloud plugin pom to point it at 8.5.

**Comment by johann-petrak** [comment]
Ah, OK, thanks! That fixes the problem (commit eb12be6b7c3432e7bd9aecbc1064d1c3e7c9d180)
I had to set it to 8.6-SNAPSHOT, the 8.5 version caused this compilation problem, so it seems
this depends on a newer version. 

```
[ERROR] /data/johann/gate-home/GIT/cloud-client/gate-plugin/src/main/java/uk/ac/gate/cloud/pr/CloudPREditor.java:[176,27] cannot find symbol
[ERROR]   symbol:   method clearAllSelections()
[ERROR]   location: class gate.gui.SerialControllerEditor
```
